TransmissionTornado.xcodeproj/project.pbxproj

```diff
@@ -41,7 +41,7 @@
 		E65A1E8E1F93F2B400762BB4 /* Server.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = Server.swift; sourceTree = "<group>"; };
 		E65A1E901F93F88B00762BB4 /* Torrent.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = Torrent.swift; sourceTree = "<group>"; };
 		E6FBD0761FB106FC00226930 /* README.md */ = {isa = PBXFileReference; lastKnownFileType = net.daringfireball.markdown; path = README.md; sourceTree = "<group>"; };
-		E6FBD07B1FB1076E00226930 /* macOS.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = macOS.app; sourceTree = BUILT_PRODUCTS_DIR; };
+		E6FBD07B1FB1076E00226930 /* TransmissionTornado.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = TransmissionTornado.app; sourceTree = BUILT_PRODUCTS_DIR; };
```

Comment:
What do you thing about the following setup:
- revert target name to **macOS**
- set the `PRODUCT_NAME` build settings, on both iOS and macOS targets, to **TransmissionTornado**

This way, the target's purpose will remain obvious, where build and product settings can be made consistent across platforms - eg. module name and application name will become ** TransmissionTornado**